tool/firebase_util.py

```diff
@@ -0,0 +1,13 @@
+import enum
```

Comment:
accidental left over from the previous version?

Reply:
Removed